week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,24 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+//Mart.random() returns a random number between 0 (inclusive),  and 1 (exclusive).
```

Comment:
Math.random() is inside the function, it's better to comment it there.
Nice comment for a function in javascript will be looking like this: [JS Doc support](https://code.visualstudio.com/docs/languages/javascript#_jsdoc-support)